subprojects/gradle/instrumentation-tests-default-config/src/main/kotlin/com/avito/instrumentation/InstrumentationDefaultConfigPlugin.kt

```diff
@@ -69,6 +69,12 @@ class InstrumentationDefaultConfigPlugin : Plugin<Project> {
                         filters.register("newUiNoE2E") {
                             it.fromSource.includeByAnnotations(TestsFilter.uiNoE2E.annotatedWith)
                         }
+                        filters.register("modifiedUi") {
```

Comment:
Should I reduce number of instrumentations? I could merge new and modified tests in one instrumentation

Reply:
Different instrumentation tasks give us ability to customize scheduling and quota settings independently.
It seems reasonable to divide these types of changes by instrumentation tasks.